spec/models/user_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  # let!(:user1) { User.create(name: 'John', posts_counter: 3) }
+  # let!(:user2) { User.create(name: 'Jane', posts_counter: 1) }
+  # let!(:user3) { User.create(name: 'Alice', posts_counter: 2) }
```

Comment:
- [ ] Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md), You are not meant to leave un-used code as a comment in your codebase. I'll suggest you remove these as leaving unused code as a comment in your codebase makes your code look a bit unprofessional :+1: :smile: :wave: